Content.Shared/Crayon/SharedCrayonComponent.cs

```diff
@@ -37,6 +37,17 @@ public CrayonColorMessage(Color color)
         }
     }
 
+    [Serializable, NetSerializable]
```

Comment:
This component has no documentation, would you mind adding some summary tags for the fields and the events?